signer/core/auditlog.go

```diff
@@ -68,6 +68,22 @@ func (l *AuditLogger) SignData(ctx context.Context, contentType string, addr com
 	return b, e
 }
 
+func (l *AuditLogger) SignGnosisTx(ctx context.Context, addr common.MixedcaseAddress, gnosisTx GnosisSafeTx, methodSelector *string) (*GnosisSafeTx, error) {
+	sel := "<nil>"
+	if methodSelector != nil {
+		sel = *methodSelector
+	}
+	l.log.Info("SignGnosisTx", "type", "request", "metadata", MetadataFromContext(ctx).String(),
+		"addr", addr.String(), "data", gnosisTx, "selector", sel)
```

Comment:
don't just dupm the `gnosisTx`, it becomes ugly in the audit log. Needs to be marshalled properly